src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page">
+    <main class="main">
+      <div class="card main__card" data-qa="card">
+        <div class="card__picture"></div>
+        <div class="card__description">
+          <p class="card__description-text">
+            Моноблок APPLE A1419 iMac<br>27" Retina 5K (MNED2UA/A)
+          </p>
+        </div>
+        <div class="card__item-code">
+          Код товара: 195434
+        </div>
+
+        <div class="review card__review">
+          <div class="stars review__stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="review__text-review">Отзывов: 5</div>
+
+          <div class="price review__price">
+            <p class="price__text">Цена:</p>
+            <p class="price__sum">69 999 грн</p>
+          </div>
+        </div>
+
+        <div class="button-buy card__button-buy">
+          <a
+            href="#"
+            class="button-buy__buy-link"
+            data-qa="hover">
+            КУПИТЬ
+          </a>
```

Comment:
Do you really need additional div here. Just use `a` tag and style it accordingly.

Also, keep an eye on indentations
```suggestion
          <a
            href="#"
            class="button-buy__buy-link"
            data-qa="hover"
          >
            КУПИТЬ
          </a>
```

Reply:
But, we can reuse the button on the next projects. So, why we can't use <div> here?

Reply:
Theoretically, yes, you are correct.
But extra `div` is still unnecessary here.